resources/js/beatmap-discussions/nominations.tsx

```diff
@@ -634,12 +634,36 @@ export class Nominations extends React.PureComponent<Props> {
             : formatDate(rankingEta)
           : trans('beatmaps.nominations.rank_estimate.soon');
 
+        const unresolvedProblems = this.props.beatmapset.discussions.filter((d) => !d.resolved && ['problem', 'suggestion'].includes(d.message_type));
+        const rankEstimatePrefixText = unresolvedProblems.length > 0
+          ? trans('beatmaps.nominations.rank_estimate.unresolved_problems')
+          : trans('beatmaps.nominations.rank_estimate.no_problems');
+
+        const problems = unresolvedProblems.map((problem) => (
```

Comment:
Should limit the maximum number of issues displayed to avoid a runoff of issues.

Reply:
hmm, then again, considering the message is for the mapper, maybe it's better to just list them all out 🤔 
I'd consider looking at a way to enable the `pending` filter to make it easier to find the discussions that need resolving, though.

Reply:
after a conversation with hivie, it seems that the warning should be separated and apply a pending filter. i'll make some changes and then resolve this

Reply:
updated. check PR description for updated functionality